modules/core/setup.php

```diff
@@ -287,9 +303,30 @@
         'junk_since' => FILTER_DEFAULT,
         'trash_per_source' => FILTER_VALIDATE_INT,
         'trash_since' => FILTER_DEFAULT,
-        'drafts_per_source' => FILTER_VALIDATE_INT,
-        'drafts_since' => FILTER_DEFAULT,
-        'warn_for_unsaved_changes' => FILTER_VALIDATE_BOOLEAN
+        'drafts_per_source' => FILTER_DEFAULT,
+        'drafts_since' => FILTER_SANITIZE_FULL_SPECIAL_CHARS,
```

Comment:
Note that we have switched from FILTER_SANITIZE_FULL_SPECIAL_CHARS to FILTER_DEFAULT for all the variables that doesn't have other specific filter preference as INTs.